Rationale.md

```diff
@@ -114,7 +114,19 @@ developer.
 
 ## Control Flow
 
-See [#299](https://github.com/WebAssembly/design/pull/299).
+Structured control flow provides simple and size-efficient binary encoding and
+compilation. Any control flow—even irreducible—can be transformed into structured
```

Comment:
This is pre-existing text that I'm just moving here, but those are Unicode em-dash characters, not hyphens, so I believe this is correct usage.